scripts/strava_auth.py

```diff
@@ -75,22 +76,29 @@ def do_GET(self):
                 access_token = token_response['access_token']
                 refresh_token = token_response['refresh_token']
                 expires_at = token_response['expires_at']
-                self.server.logger.info("Exchanged code {} for access token {}".format(code, access_token))
-                self.wfile.write(six.b("Access Token: {}\n".format(access_token)))
-                self.wfile.write(six.b("Refresh Token: {}\n".format(refresh_token)))
-                self.wfile.write(six.b("Expires at: {}\n".format(expires_at)))
-                with open('%s/strava_token.txt' % SCRIPT_DIR, 'w') as f:
-                    f.write(self.server.client_id + '\n');
-                    f.write(self.server.client_secret + '\n');
-                    f.write(access_token + '\n');
-                    f.write(refresh_token + '\n');
-                    f.write(str(expires_at) + '\n');
+                
+                self.wfile.write(six.b("{}\n".format(self.server.client_id)))
+                self.wfile.write(six.b("{}\n".format(self.server.client_secret)))
+                self.wfile.write(six.b("{}\n".format(access_token)))
+                self.wfile.write(six.b("{}\n".format(refresh_token)))
+                self.wfile.write(six.b("{}\n".format(expires_at)))
             else:
                 self.server.logger.error("No code param received.")
                 self.wfile.write(six.b("ERROR: No code param recevied.\n"))
         else:
+            SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
```

Comment:
The issue here is that it won't work as a pyinstaller bundle. I would add a command line argument instead (--server-ip).

Reply:
actually SCRIPT_DIR was already defined before in the file as:
if getattr(sys, 'frozen', False):
    # If we're running as a pyinstaller bundle
    SCRIPT_DIR = os.path.dirname(sys.executable)
else:
    SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))

that way it will work with pyinstalled bundle